### DISCUSSION TITLE
Nvim update seems broken

### DISCUSSION THREAD

**Discussion by 0xmovses** [community/community#19007] [Copilot Conversations] [answered]
I upgraded Copilot for NeoVim to latest version `1.4.2`, firstly surprised to see that `Copilot status` has been removed. 

I run `Copilot setup` authenticate and the plug does not seem to auto suggest. Any other nvim users encountering?

**Comment by tpope** [comment, marked as answer]
`:Copilot status` has not been removed. What happens when you call it?

**Reply by 0xmovses** [reply]
`unkown command 'status'`

**Reply by 0xmovses** [reply]
I just tested on VSCode and authentication works, with suggestions etc. Any idea what might be the cause here? Really would not like to use VS code!

**Reply by tpope** [reply]
Try reinstalling, maybe? I can't really think of a reason why the `status` command would disappear, which makes me think the source file got corrupted, or something of that nature.

**Reply by 0xmovses** [reply]
that worked. Thanks Tim
